examples/4_simple_storage.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
+#
+# Keybase has added an encrypted key-value store intended to support
+# security-conscious bot development with persistent state. It is a place to
+# store small bits of data that are
+#   (1) encrypted for a team or user (via the user's implicit self-team: e.g.
+# alice,alice),
+#   (2) persistent across logins
+#   (3) fast and durable.
+#
+# It supports putting, getting, listing and deleting. There is also concurrency
+# support, but check out 4_secret_storage.py for that. A team has many
+# namespaces, a namespace has many entryKeys, and an entryKey has one current
+# entryValue. Namespaces and entryKeys are in cleartext, and the Keybase client
+# service will encrypt and sign the entryValue on the way in (as well as
+# decrypt and verify on the way out) so keybase servers cannot see it or lie
+# about it.
```

Comment:
ehh. i'm comfortable with changing the language as long as we keep it as simple and readable as possible. the point here is to introduce someone to how this works so they can use it, not to provide an exhaustive discussion of tradeoffs. maybe we need a docs page that really explains the tradeoffs and design decisions, but i don't think the explanation on the simplest example should be where we go too far down that path.

Reply:
We don't need to give a lot of details, but I also want to be careful not to let someone believe the guarantees we provide are stronger than what they are actually getting. 
We don't have to do it here though, I am fine with a link to a docs page explaining it.

Reply:
i'll change "lie" to "forge" - does that address this issue?
is there a docs page yet that i can link to?
btw, since this part was somewhat copied from the design doc -- what does "durable" in "fast and durable" mean here?

Reply:
Yeah, I think forge should be fine. Not sure if we have docs on this, but I think they would be useful